app/js/components/Dashboard.jsx

```diff
@@ -0,0 +1,72 @@
+/** This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from "react";
+import { Link } from "react-router";
+
+import openmrsLogo from "../../img/openmrs-with-title-small.png";
+import loadingGif from "../../img/loading.gif";
+
+export default class Dashboard extends React.Component {
+  render() {
+    const openmrsServer = window.location.pathname.split("/")[1];
+    return (
+      <div className="list-group home-list">
+        <Link
+          to="#"
+          className="list-group-item list-group-item-action flex-column align-items-start active"
+        >
+          <div className="d-flex w-100 justify-content-between">
+            <h5 className="mb-1">The Platform is running</h5>
+            <img src={loadingGif} alt="" />
+          </div>
+        </Link>
+        <Link
+          to="#"
+          className="list-group-item list-group-item-action flex-column align-items-start"
+          onClick={event => {
+            event.preventDefault();
+            window.location = `/${openmrsServer}/owa/Add On Manager/index.html#/`;
+          }}
```

Comment:
Since this is a react app and this component is a child of React's `BrowserRouter` component, you can access `window.location` the react way via `this.props.history.location`, I advise that you do things the react way since this is a React application..

Reply:
please refactor accordingly

Reply:
There is a ticket to rework the dashboard component. I think it is better to refactor the code of that component there. Here (https://issues.openmrs.org/browse/PLAT-41) I renamed the routes and did away with the `home component naming` to `dashboard component`.